examples/pytorch/fsdp/fsdp.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
```

Comment:
Could you add some comments throughout the scripts telling what you are doing here?